app/Http/Controllers/PasswordResetController.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\ResetRequest;
+use App\Http\Requests\UpdateRequest;
+use App\Models\ResetPassword;
+use App\Services\UserService;
+use Carbon\Carbon;
+use App\Models\User;
+use Illuminate\Support\Facades\Mail;
+use App\Mail\ResetMail;
+
+class PasswordResetController extends Controller
+{
+    protected $userService;
+
+    public function __construct(UserService $userService)
+    {
+        $this->userService = $userService;
+    }
+
+    public function resetPassword(ResetRequest $request)
+    {
+        $email = $request->only('email');
+        $checkEmail = User::where('email', $email)->first();
+        if ($checkEmail)
+        {
+            $resetToken = $this->userService->createResetRow($checkEmail->id);
+            Mail::to($email)->send(new ResetMail($resetToken, $email));
+
+            return new ResetMail($resetToken, $email);
```

Comment:
why do you return 'new ResetMail'? it is the reason of your error in tests. Remove it and return null with status 200 as json.  After that go to tests and try to assert mailable sending.